chapter_5_practic/task5.cpp

```diff
@@ -0,0 +1,36 @@
+// Write a function called hms_to_secs() that takes three int values�for hours, minutes,
+// and seconds�as arguments, and returns the equivalent time in seconds(type long).
+// Create a program that exercises this function by repeatedly obtaining a time value in
+// hours, minutes, and seconds from the user(format 12:59 : 59), calling the function, and
+// displaying the value of seconds it returns.
+
+#include <iostream>
+
+inline int hms_to_secs(int h, int m, int s);
```

Comment:
why do you mark nearly all functions as intilne?

Reply:
I read it in a book) 
While this sequence of events may save memory space, it takes some extra time. There must
be an instruction for the jump to the function (actually the assembly-language instruction
CALL or something similar), instructions for saving registers, instructions for pushing arguments onto the stack in the calling program and removing them from the stack in the function
(if there are arguments), instructions for restoring registers, and an instruction to return to the
calling program. The return value (if any) must also be dealt with. All these instructions slow
down the program.
To save execution time in short functions, you may elect to put the code in the function body
directly inline with the code in the calling program. That is, each time there’s a function call in
the source file, the actual code from the function is inserted, instead of a jump to the function